plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -314,7 +314,7 @@ class Bazel(
             }
 
             Package(
-                id = it.copy(version = ""),
+                id = it,
```

Comment:
commit-message: Why is it not reasonable to always maintain the version, but only in the mentioned cases?

Reply:
With my change, the version now *is* always maintained, i.e. I've removed the only case where it was not maintained. I'll clarify that in the commit message. However, @nnobelis [argues against maintaining the version](https://github.com/oss-review-toolkit/ort/pull/9028#discussion_r1730673146). I'll reply to that separately.

Reply:
> With my change, the version now is always maintained,

Yes, I was only after reflecting this is the commit message, as you mentioned:

> i.e. I've removed the only case where it was not maintained. I'll clarify that in the commit message.

Reply:
I'm not sure whether I understand your reply correctly... is this review comment resolved now, as it was a misunderstanding originally?